test/samlp.tests.js

```diff
@@ -514,6 +514,21 @@ describe('samlp (unit tests)', function () {
       });
     });
 
+    it.only('should explode with invalid template', function(done) {
+      var samlp = new Samlp({
+        identityProviderUrl: server.identityProviderUrl,
+        requestTemplate: '<samlp:AuthnRequest xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" ID="@@ID@@" IssueInstant="@@IssueInstant@@" ProviderName="@@ProviderName@@" ProtocolBinding="@@ProtocolBinding@@" Version="2@@ HI THERE .0"><saml:Issuer xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion">@@Issuer@@</saml:Issuer></samlp:AuthnRequest>@@.0"><saml:Issuer xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion">@@Issuer@@</saml:Issuer></samlp:AuthnRequest>',
+        protocol: 'samlp',
+        providerName: 'Some name'
+      });
+
+      samlp.getSamlRequestParams({}, function(err, params) {
+        expect(err).to.be.null;
```

Comment:
Shouldn't we get back an error? Or is this not a case that reproduces the above issue?